### ISSUE TITLE
Acomodation type has 2 "amenityFeature" properties on the downloadable csv file

### ISSUE THREAD

**Issue by khalyomede** [schemaorg/schemaorg#1879]
On the file downlodable at `http://schema.org/version/latest/all-layers-types.csv`,  the type `http://schema.org/Accommodation` type has 2 properties `http://schema.org/amenityFeature`. 

Is this an actual bug or it this intentional?

**Comment by RichardWallis** [comment]
Well spotted @khalyomede 

After some digging it seems that [amenityFeature](http://schema.org/amenityFeature) was unnecessarily defined as a property for [Accommodation](http://schema.org/Accommodation), as it is already defined for [Place](http://schema.org/Place) from which it inherits.

I will register this as a defect as it at least should have raised a warning or similar.

**Comment by RichardWallis** [comment]
Candidate for a new test to identify the unnecessary declaration of a property for a type, where the property is already inherited from a supertype.

**Comment by RichardWallis** [comment]
Built new test to check for unnecessary domainIncludes statements. ie. Where a property is defined for a type, that is already defined in, and therefore inherited from, a supertype.

Results show 31 errors:
```
test_unnecessaryDomainIncludes (test_graphs.SDOGraphSetupTestCase) ... INFO:test_graphs:Unnecessary Domain Includes errors!!!

INFO:test_graphs:Property 'actionableFeedbackPolicy' has unneccesary domainInclude of type 'NewsMediaOrganization' - aready inherited from 'Organization'
INFO:test_graphs:Property 'activeIngredient' has unneccesary domainInclude of type 'DietarySupplement' - aready inherited from 'Substance'
INFO:test_graphs:Property 'activeIngredient' has unneccesary domainInclude of type 'Drug' - aready inherited from 'Substance'
INFO:test_graphs:Property 'alumni' has unneccesary domainInclude of type 'EducationalOrganization' - aready inherited from 'Organization'
INFO:test_graphs:Property 'amenityFeature' has unneccesary domainInclude of type 'Accommodation' - aready inherited from 'Place'
INFO:test_graphs:Property 'amenityFeature' has unneccesary domainInclude of type 'LodgingBusiness' - aready inherited from 'Place'
INFO:test_graphs:Property 'cause' has unneccesary domainInclude of type 'MedicalSignOrSymptom' - aready inherited from 'MedicalCondition'
INFO:test_graphs:Property 'codeValue' has unneccesary domainInclude of type 'MedicalCode' - aready inherited from 'CategoryCode'
INFO:test_graphs:Property 'correctionsPolicy' has unneccesary domainInclude of type 'NewsMediaOrganization' - aready inherited from 'Organization'
INFO:test_graphs:Property 'diversityPolicy' has unneccesary domainInclude of type 'NewsMediaOrganization' - aready inherited from 'Organization'
INFO:test_graphs:Property 'diversityStaffingReport' has unneccesary domainInclude of type 'NewsMediaOrganization' - aready inherited from 'Organization'
INFO:test_graphs:Property 'duration' has unneccesary domainInclude of type 'Audiobook' - aready inherited from 'MediaObject'
INFO:test_graphs:Property 'ethicsPolicy' has unneccesary domainInclude of type 'NewsMediaOrganization' - aready inherited from 'Organization'
INFO:test_graphs:Property 'healthCondition' has unneccesary domainInclude of type 'Patient' - aready inherited from 'PeopleAudience'
INFO:test_graphs:Property 'indication' has unneccesary domainInclude of type 'TherapeuticProcedure' - aready inherited from 'MedicalProcedure'
INFO:test_graphs:Property 'isAccessibleForFree' has unneccesary domainInclude of type 'PublicationEvent' - aready inherited from 'Event'
INFO:test_graphs:Property 'legalStatus' has unneccesary domainInclude of type 'Drug' - aready inherited from 'MedicalEntity'
INFO:test_graphs:Property 'legalStatus' has unneccesary domainInclude of type 'DietarySupplement' - aready inherited from 'MedicalEntity'
INFO:test_graphs:Property 'maximumIntake' has unneccesary domainInclude of type 'Drug' - aready inherited from 'Substance'
INFO:test_graphs:Property 'maximumIntake' has unneccesary domainInclude of type 'DietarySupplement' - aready inherited from 'Substance'
INFO:test_graphs:Property 'medicalSpecialty' has unneccesary domainInclude of type 'Hospital' - aready inherited from 'MedicalOrganization'
INFO:test_graphs:Property 'medicalSpecialty' has unneccesary domainInclude of type 'Physician' - aready inherited from 'MedicalOrganization'
INFO:test_graphs:Property 'medicalSpecialty' has unneccesary domainInclude of type 'MedicalClinic' - aready inherited from 'MedicalOrganization'
INFO:test_graphs:Property 'numberOfRooms' has unneccesary domainInclude of type 'House' - aready inherited from 'Accommodation'
INFO:test_graphs:Property 'numberOfRooms' has unneccesary domainInclude of type 'Apartment' - aready inherited from 'Accommodation'
INFO:test_graphs:Property 'numberOfRooms' has unneccesary domainInclude of type 'SingleFamilyResidence' - aready inherited from 'Accommodation'
INFO:test_graphs:Property 'numberOfRooms' has unneccesary domainInclude of type 'Suite' - aready inherited from 'Accommodation'
INFO:test_graphs:Property 'numberOfRooms' has unneccesary domainInclude of type 'SingleFamilyResidence' - aready inherited from 'House'
INFO:test_graphs:Property 'ownershipFundingInfo' has unneccesary domainInclude of type 'NewsMediaOrganization' - aready inherited from 'Organization'
INFO:test_graphs:Property 'possibleTreatment' has unneccesary domainInclude of type 'MedicalSignOrSymptom' - aready inherited from 'MedicalCondition'
INFO:test_graphs:Property 'productionDate' has unneccesary domainInclude of type 'Vehicle' - aready inherited from 'Product'
INFO:test_graphs:Property 'purchaseDate' has unneccesary domainInclude of type 'Vehicle' - aready inherited from 'Product'
INFO:test_graphs:Property 'unnamedSourcesPolicy' has unneccesary domainInclude of type 'NewsMediaOrganization' - aready inherited from 'Organization'
```

Question now arrises -- do we fix all these now, or leave them in tests as 'expected failures' for later fixing?

/cc @danbri

**Comment by RichardWallis** [comment]
Released in V3.4
